invenio_rdm_records/records/jsonschemas/records/record-v3.0.0.json

```diff
@@ -226,7 +223,6 @@
           "$comment": "This part of the schema is modelled on GeoJSON, but without the scope to embed arbitrary metadata.\n\nWe nest `features` within `locations` to give scope later to say something about the locations as a whole.",
           "type": "object",
           "additionalProperties": false,
-          "required": ["features"],
```

Comment:
might this be the cause of the issue? I do not see features anywhere in marshmallow...